source/node/Descriptor.js

```diff
@@ -79,27 +77,41 @@ module.exports = function describe(dataset, parentGroupID) {
     }
     entries.forEach(function(entry) {
         commands.push(
-            Inigo.create(Commands.CreateEntry)
+            Inigo.create(Command.CreateEntry)
                 .addArgument(currentGroupID)
                 .addArgument(entry.id)
                 .generateCommand()
         );
+        // Deprecated: this will soon be removed (v3):
         ["title", "username", "password"].forEach(function(property) {
             if (entry[property]) {
                 commands.push(
-                    Inigo.create(Commands.SetEntryProperty)
+                    Inigo.create(Command.SetEntryProperty)
                         .addArgument(entry.id)
                         .addArgument(property)
                         .addArgument(entry[property])
                         .generateCommand()
                 );
             }
         });
+        if (entry.properties) {
+            for (let propertyName in entry.properties) {
```

Comment:
This could be written simpler as:

```js
Object.keys(entry.properties).forEach(prop => {
	commands.push(...);
});
```

This works too, but that way you don't have to use `hasOwnProperty`